### PR TITLE
Headers for TS files

### PR DESCRIPTION
**Pull request by mikaelm12**
Issue: https://github.com/aspnet/SignalR/issues/797

### CODE REVIEW

File: client-ts/Microsoft.AspNetCore.SignalR.Client.TS/gulpfile.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
-const gulp       = require('gulp');
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (moozzyk) [inline client-ts/Microsoft.AspNetCore.SignalR.Client.TS/gulpfile.js]: I don't think we include headers in build files

**Comment by moozzyk** [comment]
Do we need to add headers to tests?

**Comment by muratg** [comment]
@moozzyk We've been doing that for the other repos, so I think we should do it here too.

**Comment by mikaelm12** [comment]
Yeah, the TS test project was showing up for me in Visual Studio so I missed it. Added them now though

File: client-ts/Microsoft.AspNetCore.SignalR.Client.TS/gulpfile.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const gulp       = require('gulp');
```
Comment (moozzyk) [inline client-ts/Microsoft.AspNetCore.SignalR.Client.TS/gulpfile.js]: revert

**Comment by moozzyk** [comment]
There is one more test suite: https://github.com/aspnet/SignalR/tree/dev/client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/js

**Comment by muratg** [comment]
:shipit:

**Pull request merged**